src/device/usbd.c

```diff
@@ -37,6 +37,8 @@
 #define CFG_TUD_TASK_QUEUE_SZ   16
 #endif
 
+#define ARRAY_LENGTH(ARR) (sizeof(ARR) / sizeof(ARR[0]))

```

Comment:
Perhaps unnecessary. Can remove if desired (or move to a more global header as TU_ARRAY_LENGTH.

Reply:
there is already TU_ARRAY_SIZE() macro in the tusb_common.h file :)